homework/hackyourtemperature/server.js

```diff
@@ -0,0 +1,18 @@
+import express from 'express';
+// import { engine } from 'express-handlebars';
+// import fetch from 'node-fetch';
+
+const app = express();
+
+app.use(express.json());
+
+app.get('/', (req, res) => {
+  res.send('Hello from backend to frontend!');
+});
+
+app.post('/weather', (req, res) => {
+  const cityName = req.body.cityName;
+  res.json(cityName);
```

Comment:
I'm sorry didn't notice the `res.send` thought it was `res.json` my bad

```suggestion
  res.json({ cityName });
```

Now this should be okay with the guidelines and should work normally